hw/ip/otp_ctrl/doc/_index.md

```diff
@@ -574,6 +574,11 @@ In case of a mismatch, the buffered values are gated to their default, and an al
 
 Note that in case of unrecoverable OTP errors or ECC failures in the buffer registers, the partition controller FSM is moved into a terminal error state, which locks down all access through DAI and clamps the values that are broadcast in hardware to their defaults.
 
+External escalation via the `lc_escalate_en` signal will move the partition controller FSM into the terminal error state as well, except for the case where the controller FSM belongs to the LIFE_CYCLE partition.
+The reason for that is the separation of the "wipe secrets" escalation action (distributed via the `lc_escalate_en` signals) and the "life cycle state scrapping" escalation action, which is implemented inside the life cycle controller.
+Invalidating the life cycle state here via `lc_escalate_en` would interfere with that separation, which allows to schedule said actions at different times in the escalation protocol.
```

Comment:
Could be my bad english -> what is a "said actions"?

Reply:
oh this could be rephrased as "the actions mentioned before". let me change that a bit.

Reply:
Oh I got it!